annlite/index.py

```diff
@@ -634,11 +634,10 @@ def dump_model(self):
 
     def dump_index(self):
         logger.info(f'Save the indexer to {self.index_path}')
-
         try:
             self.index_path.mkdir(parents=True)
 
-            self.meta_table.dump(self.index_path / 'meta_table.db')
+            self.meta_table.dump(self.index_path / 'meta.db')
```

Comment:
this is not needed, right?

Reply:
in `table.py` we use `self._conn_name = data_path / f'{name}.db'`.

Reply:
I mean the `dump()` step is not needed caused meta_table is file-based already.